content/java/concepts/strings/terms/equalsIgnoreCase/equalsIgnoreCase.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: '.equalsIgnoreCase()'
+Description: 'Compares strings in a case-insensitive manner.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'String'
+CatalogContent:
+  - 'learn-java'
+---
+
+The **`.equalsIgnoreCase()`** method compares strings and then then gives an output of true or false. It is not a case sensitive method.
+
+## Syntax
+
+```pseudo
+boolean result = stringOne.equalsIgnoreCase(stringTwo);
+```
+On the syntax above, we have three different variables. One of them being a boolean type variable, the other two being a string type variable. As mentioned on the description of this method, `.equalsIgnoreCase()` compares strings in a case insensitive way, so if the values stored in the strings are same, the result stored in `result` will be true. Otherwise, it will be false.

```

Comment:
```suggestion
In the syntax above, there are three different variables. One of them, `result` being a boolean type variable, the other two being string type variables. If the values stored in the strings are same, ignoring the case, the result stored in `result` will be true. Otherwise, it will be false.

```